docs/dataset-overviews/amip-sst-sea-ice-boundary-forcing.md

```diff
@@ -21,7 +21,7 @@ The source ID that identifies the dataset to use in the different phases of CMIP
 
 #### Testing
 
-No data available for this phase yet.
+For the testing phase of CMIP7, use data with the source ID [PCMDI-AMIP-1-1-9](https://aims2.llnl.gov/search?project=input4MIPs&versionType=all&&activeFacets=%7B%22source_id%22%3A%5B%22PCMDI-AMIP-1-1-9%22%5D%7D)
```

Comment:
The AR7 Fast Track can use PCMDI-AMIP-1-1-9, coverage is provided to 2022-12, so gets us across the line, with a completely new dataset hopefully landing mid-year (I don't want to write that on pages, it's taken long enough to get HadISST data up to 2021..)

Reply:
Nice, thanks